b0tweb/FileHelper.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.IO;
+using System.Reflection;
+
+namespace b0tweb
+{
+    /// <summary>
+    /// FileHelper implements general file operations we need in the b0tweb.
+    /// </summary>
+    class FileHelper
+    {
+        public static string GetBasePath()
+        {
+            return Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location);
+        }
```

Comment:
Documentation is missing

Reply:
![image](https://user-images.githubusercontent.com/32071671/75879250-5a47bc00-5e1b-11ea-9f0a-938f6cb8de32.png)